### ISSUE TITLE
any hints on manual cf worker deployment?

### ISSUE THREAD

**Issue by wizzard0** [fab-spec/fab#166]
first, FAB is an awesome idea which bugged me since I heard about CF workers (even before KV/Sites, as you could still package a small website w/routing logic inside the worker body!)

Now, in practice, I'm stuck trying to deploy the FAB I've got :)

I understand the automatic deployer is still in progress, but Workers are mentioned everywhere as the prime target, so I guess it should be possible with a bit of manual work :)

The link to v0 docs gives 'Internal Server Error', unfortunately :(
https://fab-docs--production--v0.branch.linc-preview.sh/

So... any hints? Thanks in advance!

**Comment by geelen** [comment]
Hey there, docs are still a WIP I'm afraid, getting the deploy instructions out is the biggest blocking factor before I can release. The v0 deployer stuff is not compatible at all with FAB v1, sorry.

If you want to release automatically, you can use Linc: https://linc.sh/docs/cloudflare-workers

Basically, as long as your app is on GitHub and has a `build:fab` step, Linc can handle the auto-publishing to Workers just fine. It's just the command-line stuff that's undocumented.

**Comment by wizzard0** [comment]
OK, will try right now

**Comment by wizzard0** [comment]
...ok, static-on-linc seems to work, now for the API part... I guess we should continue this convo elsewhere :)

**Comment by geelen** [comment]
Yep, but for reference (and for future people reading), this is the page with documentation for adding server-side API stuff to a FAB: https://fab.dev/guides/adding-server-side-logic
